### DISCUSSION TITLE
[Question]: HA shows my binary sensors for frigate constantly flapping to unavailable

### DISCUSSION THREAD

**Discussion by cchance27** [blakeblackshear/frigate#11367] [Ask A Question] [answered]
### What is your question:

I'm not sure when this started but its definitly not good, i can watch the log page and see it happening second to second and seems to be happening on every frigate sensor.

<img width="659" alt="image" src="https://github.com/blakeblackshear/frigate/assets/13766359/f460249f-e6fd-4298-9b76-f106fe7b13f3">

**Comment by NickM-27** [comment, marked as answer]
This usually is an issue when multiple frigate instances are installed and running.

**Reply by cchance27** [reply]
Your a genius, i had migrated my frigate from docker to running in an LXC on my proxmox, and low and behold after a reboot today the damn docker instance started back up :) Thanks
